bika/lims/content/samplepoint.py

```diff
@@ -79,7 +77,7 @@
         allowed_types=('SampleType',),
         vocabulary='SampleTypesVocabulary',
         relationship='SamplePointSampleType',
-        widget=brw(
+        widget=ReferenceWidget(
```

Comment:
`bika.lims.browser.widgets.referencewidget.ReferenceWidget` (https://github.com/senaite/bika.lims/blob/master/bika/lims/browser/widgets/referencewidget.py) provides the search overlay that, for instance, is used in SampleType's edit view for the selection of multiple Sample Points, as well as in a lot of different places (e.g. Add AR). I'd expect the same behavior in both places, and honestly, I like the search overlay more than the multiselection list.... 

Could we tweak our ReferenceWidget to fix this issue instead of using `Products.Archetypes.public.ReferenceWidget`?

Reply:
Good catch, actually I forgot about the edit capabilities of the referencewidget.
I'll fix `bika.lims.browser.widgets.referencewidget.ReferenceWidget` to handle the view case better for old-style references.